src/client/sandbox/node/element.ts

```diff
@@ -454,6 +454,13 @@ export default class ElementSandbox extends SandboxBase {
         const sandbox = this;
 
         this.overriddenMethods = {
+            appendData (...args) {
```

Comment:
as far as I understand we do not need args array here since we use only args[0] parameter, so I think we can replace `...args` with `text`